docs/examples/frameworks/tensorflow/tensorflow-dataset-multigpu.ipynb

```diff
@@ -178,7 +177,7 @@
     "                    BATCH_SIZE, device_id=i, shard_id=i, num_shards=NUM_DEVICES),\n",
     "                batch_size=BATCH_SIZE,\n",
     "                output_shapes=shapes,\n",
-    "                output_dtypes=dtypes,\n",
+    "                dtypes=dtypes,\n",
```

Comment:
This is specification of dataset, it uses `output_dtypes` (to mimic the TF api).

Reply:
Right, I thought I skipped those but in this one I changed it by mistake. Good catch